.gitignore

```diff
@@ -1,5 +1,10 @@
 target/
+Queue.jar
+GenomeAnalysisTK.jar
```

Comment:
There is an dependency in lib/sis-jhdf5-batteries_included.jar that can't be downloaded from Maven central repository. It will be ignored if we use *.jar. Anyway, I have a fix for it by downloading and preparing it from jhdf5 homepage.